main.py

```diff
@@ -195,27 +195,51 @@ async def inner_f():
 
         # Wait some time between songs
         if seconds_between_songs:
-            await current_channel.send(
-                f"Chillin' for {seconds_between_songs} seconds..."
+            embed_title = "Currently Chillin'"
+            embed_content = "Waiting for {} second{}...".format(
+                seconds_between_songs, "s" if seconds_between_songs != 1 else ""
             )
+            embed = discord.Embed(title=embed_title, description=embed_content)
```

Comment:
Just checking it was intentional to not include a color for this embed.

Personally I think this one doesn't need a color or need to be too loud, and it's just supposed to be a short transitional message while people get their last words in.

Reply:
Yup! it was intentional